src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -139,6 +139,16 @@ protected function p(Node $node, $parentFormatPreserved = false): string
             : $content;
     }
 
+    protected function pAttributeGroup(Node\AttributeGroup $node): string
+    {
+        $ret = '#[' . $this->pCommaSeparated($node->attrs) . ']';
```

Comment:
original `parent::pAttributeGroup()` should be called first imo

Reply:
Good shout, updated